src/request.ts

```diff
@@ -1044,23 +1038,22 @@ class DatastoreRequest {
       mutations,
     };
 
-    function onCommit(err: Error|null, resp: {mutationResults: Entity;}) {
+    function onCommit(err?: Error|null, resp?: {mutationResults: Entity;}) {
       if (err || !resp) {
         callback(err, resp);
         return;
       }
 
-      arrify(resp.mutationResults)
-          .forEach((result: Entity, index: number) => {  //! Entity malo
-            if (!result.key) {
-              return;
-            }
+      arrify(resp.mutationResults).forEach((result: Entity, index: number) => {
```

Comment:
why do we need explicit types here?  Can we dump `: Entity` and `:number`

Reply:
`number` yes, `result` is implicitly being assigned `any` 😅